packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +135,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? new ClampingScrollPhysics(),
```

Comment:
We need the clamping physics to make nested scroll view work. If you turn off the clamping, it'll go crazy.